frontend/package.json

```diff
@@ -12,8 +12,10 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "5.0.0",
-    "web-vitals": "^2.1.3"
+    "react-scripts": "^2.1.3",
```

Comment:
We were working on version 5.0.0. Not sure how it got downgraded to 2.1.3. Change this to `"react-scripts": "5.0.0",` and we'll see if it causes any issues